src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -21,6 +21,24 @@ const HEALTH_TOOLS = [
   frontendApps.APPOINTMENTS,
 ];
 
+const MHV_ACCOUNT_TYPES = ['Premium', 'Advanced', 'Basic'];
+
+export const hasRequiredMhvAccount = (appId, accountLevel) => {
+  switch (appId) {
+    case frontendApps.HEALTH_RECORDS:
+    case frontendApps.LAB_AND_TEST_RESULTS:
```

Comment:
Included this small change to fix department-of-veterans-affairs/vets.gov-team#14187.